src/blockdev.rs

```diff
@@ -322,18 +321,23 @@ impl Partition {
         // We multiply by 512 here: the kernel values are always in 512 blocks, regardless of the
         // actual sector size of the block device. We keep the values as bytes to make things
         // easier.
-        let start_offset: u64 = start.checked_mul(512).ok_or("start offset mult overflow")?;
+        let start_offset: u64 = start
+            .checked_mul(512)
+            .ok_or_else(|| anyhow!("start offset mult overflow"))?;
```

Comment:
Hmm, interesting. Does `ok_or("string")` not work anymore? I would think the implied `into()` would convert it fine.

Reply:
It doesn't.  `anyhow::Error` doesn't implement `From<&str>`.

Reply:
Ack.